src/Symfony/Component/Form/Extension/Core/Type/IntegerType.php

```diff
@@ -36,9 +37,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $scale = function (Options $options) {
+            if (null !== $options['precision']) {
```

Comment:
`isset($options['precision'])` would make the check work even when the key does not exist